packages/mgt-components/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -537,17 +535,8 @@ export class MgtAgenda extends MgtTemplatedComponent {
       return 'ALL DAY';
     }
 
-    // #937 When not specifying a preferred time zone using the
-    // preferred-timezone attribute, MGT treats the dates retrieved from
```

Comment:
We should restore this chunk of logic here to force the dateTimeString to be in UTC